docs/usage/cli.md

```diff
@@ -0,0 +1,35 @@
+# Command Line Interface (CLI)
```

Comment:
The autogenerated arguments as it renders on readthedocs don't seem to include the arguments defined in the config. Those are really important, and I think it's worth going through them explicitly anyway. (i.e. `--overlay_bids_dir`, `--base_desc`, `--overlay_desc`, and the exceptions and drops args)

Reply:
Otherwise can manually write those for here

Reply:
Yeah, you will need to add the `add_dynamic_args` in if it isn't already there.